src/Footer.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Footer = () => {
+  const contacts = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
+  return (
```

Comment:
The instructions said to pass the data as a prop - that's preferable as I mentioned above